fetch.js

```diff
@@ -93,60 +112,15 @@
   }
 
   Headers.prototype.keys = function() {
-    var items = []
-    this.forEach(function(value, name) { items.push(name) })
-    var iterator = {
-      next: function() {
-        var value = items.shift()
-        return {done: value === undefined, value: value}
-      }
-    }
-
-    if (support.iterable) {
-      iterator[Symbol.iterator] = function() {
-        return iterator
-      }
-    }
-
-    return iterator
+    return getIterator(this, function(value, name) { this.push(name) })
```

Comment:
No I think the comment is too much. If someone is interested in this, whey will inspect the source. It's also now weird that the comment is at the 1st invocation of this function but not on the other ones. Please revert the comment.